FF1Lib/FF1Rom.cs

```diff
@@ -1507,5 +1514,18 @@ public void ShuffleRng(MT19337 rng)
 			Put(BattleRngOffset, battleRng.SelectMany(blob => blob.ToBytes()).ToArray());
 		}
 
+		private void StampRom(String _seed, String _encodedFlagString, String _commitSha, String _inputRomSha){
+			RomStampModel stamp = new RomStampModel();
```

Comment:
I _think_ you should be able to initialize this inline which will make it a little nicer: https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/how-to-initialize-objects-by-using-an-object-initializer